script/Ciel/Day 13/13_01_01_CIEL13_2.txt

```diff
@@ -387,17 +411,19 @@ Even though it's already dead, this lump of flesh is pretending to be alive, and
 [sha:b5c5c04f329ab00a006689e35a5574dac03dd509]{
 -- Page 94, Offset 33420.
 -- 自分でも聞き取れない叫び声を上げて走る。
-I let out a scream that even I can't comprehend before running away.
+Deaf to my own scream, I break into a run.
```

Comment:
Somehow the "I break into a run" bothers me. It sorta takes over the sentence when it's just a tiny little addendum in the JP one. It being at the end and leading into the next line is good though, especially if they're the same conjugation.

>I let out a scream that I can't even comprehend myself, and run.